docs/README.md

```diff
@@ -193,34 +193,33 @@ The version of sphinx theme is fixed in both repositories:
 * https://github.com/apache/airflow-site/blob/main/sphinx_airflow_theme/sphinx_airflow_theme/__init__.py#L21
 * https://github.com/apache/airflow/blob/main/devel-common/pyproject.toml#L77 in "docs" section
 
-In case of bigger changes to the theme, we
-can first iterate on the website and merge a new theme version, and only after that we can switch to the new
+In case of bigger changes to the theme, we can first iterate on the website and merge a new theme version, and only after that can we switch to the new
```

Comment:
```suggestion
In case of bigger changes to the theme, we can first iterate on the website and merge a new theme version,
and only after that can we switch to the new
```

Reply:
we don't want the line to be super long